<launch>
    <node pkg="multibotnet" type="multibotnet_node" name="multibotnet_node" output="screen">
        <param name="config_file" value="$(find multibotnet)/config/default.yaml"/>
    </node>
</launch>